mission_control/navigator_launch/launch/master_shore.launch

```diff
@@ -1,3 +1,9 @@
 <launch>
```

Comment:
I like what we were doing for the sub where we would extract all of these little params to yaml config files all in the same launch/config directory. What do you think?